src/components/NoConnectionIndicator/NoConnectionIndicator.tsx

```diff
@@ -29,7 +29,7 @@ const NoConnectionIndicator: React.FC<NoConnectionIndicatorProps> = ({
         </IconWrapper>
         <TextWrapper>
           {!isConnectedToInternet ? (
-            <Text variant="subtitle2">No network connection</Text>
+            <Text variant="subtitle2">Network connection lost</Text>
```

Comment:
I think here we can leave the old copy. The difference is that snackbar tells you about something that happened, so "Network connection was lost". But this indicator is more permanent and it tells you about current status, so "No network connection"